include/dfu/suit_dfu_fetch_source.h

```diff
@@ -65,37 +72,38 @@ typedef int (*suit_dfu_fetch_source_request_fn)(const uint8_t *uri, size_t uri_l
 /**
  * @brief Register fetch source
```

Comment:
```suggestion
 * @brief Register a fetch source
```
Without an article, I read this sentence as a verb-verb-noun that makes no sense. With an article I see it's verb-adjective-noun